application/controllers/ajax/admin/removeFile.php

```diff
@@ -7,6 +7,8 @@ public function get_index($id = null)
 			$user = $file->getUser();
 			if(!empty($file) && !empty($_SESSION['connected_user_id']) && ($_SESSION['connected_user_id'] == $user->prop('id') || $_SESSION['connected_user_group'] === 2))
```

Comment:
Hm, n'a-t-on pas accès ici à `$this->_current_member` ? S'il est empty, c'est que le membre n'est pas connecté, sinon, c'est qu'il l'est. Si on y a bien accès, il serait mieux de l'utiliser plutôt que les variables de sessions :)

De plus, à quoi correspond ce `2` pour le groupe ? Il faut éviter les nombres magiques pour un code plus clair :)